package-lock.json

```diff
@@ -25973,9 +25973,9 @@
       "devOptional": true
     },
     "async": {
-      "version": "2.6.3",
-      "resolved": "https://registry.npmjs.org/async/-/async-2.6.3.tgz",
-      "integrity": "sha512-zflvls11DCy+dQWzTW2dzuilv8Z5X/pjfmZOWba6TNIVDm+2UDaJmXSOXlasHKfNBs8oo3M0aT50fDEWfKZjXg==",
+      "version": "2.6.4",
+      "resolved": "https://registry.npmjs.org/async/-/async-2.6.4.tgz",
+      "integrity": "sha512-mzo5dfJYwAn29PeiJ0zvwTo04zj8HDJj0Mn8TD7sno7q12prdbnasKJHhkm2c1LgrhlJ0teaea8860oxi51mGA==",
```

Comment:
It looks like you modified the package-lock directly as opposed to setting overrides at the package level.
I'm unfamiliar with this pattern and not entirely sure what possible consequences of this may be, but I can't perceive any issues with it at the moment

Reply:
Wouldn't this be problematic as soon as another update is make to `package.json` and the lock file is regenerated?

Reply:
@mcmcphillips Yeah I had the vulnerability issue before, and npm audit fix didn't fix the issue. So I tried to manually change it, as suggested.

I just talked to Matt, and he is ok to leave it as is so I'll resolve the conversation.